src/main/g8/project/Dependencies.scala

```diff
@@ -9,14 +9,14 @@ object Dependencies {
 
     // Scala third-party
     val catsCore   = "1.6.1"
-    val catsEffect = "1.4.1"
-    val circe      = "0.11.1"
+    val catsEffect = "1.4.0"
```

Comment:
It should be 2.x. Circe 0.13.0 depends on Cats 2, as well as Cats Effect, so we have incompatible versions. Also I'd probably remove cats core altogether because it just won't represent the actual dependency tree.

Reply:
Done.